### ISSUE TITLE
Unable to join the server (packet limit)

### ISSUE THREAD

**Issue by Lgmrszd** [TeamAOF/All-of-Fabric-6#118]
After updating the server to 1.4.3 (was 1.3.0) it is now impossible to connect to the server, player gets kicked.
Server doesn't crash however this error appears in the log:
```log
io.netty.handler.codec.EncoderException: String too big (was 34278 characters, max 32767)
...
at uk.me.desert_island.rer.RoughlyEnoughResources.writeJsonPrimitive(RoughlyEnoughResources.java:140) ~[rer-2.6.0.jar:?]
```
Full traceback from the log: https://gist.github.com/Lgmrszd/94caf5330a838c40f367b4581c441ed4

Seems Roughly Enough Resources is the issue:
https://github.com/theorbtwo/RoughlyEnoughResources/issues/60

**Comment by Linguardium** [comment]
yep, and someone made a fix back in Aug and apparently it hasnt been included in the mod https://github.com/theorbtwo/RoughlyEnoughResources/pull/69

**Comment by Lgmrszd** [comment]
Temporarily solution for that is to remove the mod on a server, clients don't need to remove it in order to join.

**Comment by Lgmrszd** [comment]
Is it fixed in dev or in 1.5.3? I still experience this issue on 1.5.3.
